core/loadpoint.go

```diff
@@ -1082,42 +1083,71 @@ func (lp *Loadpoint) updateChargerStatus() (bool, error) {
 
 	lp.log.DEBUG.Printf("charger status: %s", status)
 
+	// detect status changes
```

Comment:
This whole part needs a cleanup and deduplication:
- identify status change (either duration or status)
- process the change
- publish updated current

In this state, there is too much duplicated code that will be hard to maintain